hack/allure-report.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash -ex
+
+CONTAINER_RUNTIME=${CONTAINER_RUNTIME:-podman}
```

Comment:
is this line needed now?

Reply:
It is for the case you want to run the script standalone